### PR TITLE
Add handwired/pteron config and keymaps

### PR DESCRIPTION
**Pull request by FSund**
I would like to add my handwired keyboard "*pteron*" to the QMK keyboards.

Some background information on the keyboard can be found in the [github repo where the case files are hosted](https://github.com/FSund/pteron-keyboard), and in the accompanying [Keebtalk](https://www.keebtalk.com/t/designing-an-atreus-iris-inspired-keyboard-the-pteron/2686/) thread.

I have only made a config for the default 56key layout for now, but it should be easy to adapt to the smaller versions.

### CODE REVIEW

File: keyboards/handwired/pteron/config.h [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#ifndef CONFIG_H
```
Comment (drashna) [inline keyboards/handwired/pteron/config.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

File: keyboards/handwired/pteron/config.h [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+// #define MANUFACTURER    qmkbuilder
```
Comment (drashna) [inline keyboards/handwired/pteron/config.h]: This still needs to be there.  
If you want "QMK Community" is a good option for this.

File: keyboards/handwired/pteron/keymaps/FSund/keymap.c [hunk 3]
```diff
@@ -0,0 +1,119 @@
+#include "pteron.h"
```
Comment (drashna) [inline keyboards/handwired/pteron/keymaps/FSund/keymap.c]: Could you replace this include with `#include QMK_KEYBOARD_H` instead? 

And the action_layer include is not needed, and should be removed.

File: keyboards/handwired/pteron/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,105 @@
+#include "pteron.h"
```
Comment (drashna) [inline keyboards/handwired/pteron/keymaps/default/keymap.c]: Could you replace this include with `#include QMK_KEYBOARD_H` instead? 

And the action_layer include is not needed, and should be removed.

File: keyboards/handwired/pteron/readme.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+Keyboard Maintainer: [Filip Sund](https://github.com/FSund)  
+Hardware Supported: Pro Micro  
+Hardware Availability: Handwired, no PCB's available (for now)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make handwired/pteron:default
+
+See [build environment setup](https://docs.qmk.fm/build_environment_setup.html) then the [make instructions](https://docs.qmk.fm/make_instructions.html) for more information.
```
Comment (drashna) [inline keyboards/handwired/pteron/readme.md]: Could you replace this link block with this, instead: 

```markdown
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

File: keyboards/handwired/pteron/rules.mk [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+# Bootloader
+#     This definition is optional, and if your keyboard supports multiple bootloaders of
+#     different sizes, comment this out, and the correct address will be loaded
+#     automatically (+60). See bootloader.mk for all options.
+BOOTLOADER = caterina
+
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE ?= no	# Virtual DIP switch configuration(+1000)
```
Comment (drashna) [inline keyboards/handwired/pteron/rules.mk]: Could you remove all of the question marks here?  They're not needed.

**Comment by FSund** [comment]
I've done all the changes you mentioned (and replaced an include guard you missed). Could you have a look at it?

Thanks a lot for all your work with the QMK repo!

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**